apps/cron/src/tasks/processGemsPayout/index.ts

```diff
@@ -60,6 +63,19 @@ export async function processGemsPayout(ctx: Context, { now = DateTime.utc() }:
     }
   }
 
+  try {
+    const [newScoutRewards, topReferrerRewards] = await Promise.all([
+      createNewScoutRewardsContract({ week, season }),
+      createTopReferrerRewardsContract()
+    ]);
+    log.info('Rewards contract deployed', {
+      newScoutRewards,
+      topReferrerRewards
+    });
+  } catch (error) {
+    log.error('Error deploying new scout contract', { error });
```

Comment:
i would log week and season here too since those are inputs to one of the methods